tests/plugins/clipboard/pastefakeelement.js

```diff
@@ -0,0 +1,89 @@
+/* bender-tags: editor */
```

Comment:
It'd be nice to have mentioned breakable cases tested here.

Reply:
There is also one more issue with unit tests: they pass even without the fix – as whole pasting is treated as from the same page/tab.